acceptance-tests.md

```diff
@@ -124,20 +146,25 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Generate and share operations alerts**
     * _As an administrator of the cluster, I should be able to generate and share operation alerts in OpenShift using the monitoring tools available in the NERC cluster._
-    * <span style="text-decoration:underline;">Steps</span>
-        1. Alerts are sent to Slack
+    * Steps
+        1. Adding alerts to Slack involves creating a pull request with the alert name, summary, description, expression, for, and labels in the `thanos-ruler-custom-rules` ConfigMap. It also requires adding the corresponding alert names to the matchers in the `open-cluster-management-observability-alertmanager-config` ExternalSecret. [Click here to see an example pull request of adding custom alerts to Slack](https://github.com/OCP-on-NERC/nerc-ocp-config/pull/103).
+    * Acceptance tests:
+        1. [Click here to check that alerts are sent to `alerts-nerc-ocp` channel in Slack](https://app.slack.com/client/T14CVG7L1/C044ZCTUWCV/thread/C027TDE52TZ-1666641224.695039).
 2. **Logging**
     * _As an administrator of the cluster, I should be able to track all the events in the cluster using the logging system in OpenShift._
```

Comment:
Log archiving and rollover is something that often makes acceptance tests fail. after systems have been running for a while  Please document how to check on log storage space consumed vs. available using appropriate dashboards or commands.

Reply:
@computate This documentation still needs to be added.